src/nashpy/algorithms/lemke_howson_lex.py

```diff
@@ -12,28 +12,28 @@
 
 def lemke_howson_lex(A, B, initial_dropped_label=0):
     """
-	Obtain the Nash equilibria using the Lemke Howson algorithm implemented
```

Comment:
This is a fix that I think black picked up for you. 

It's a great fix and fits with an idea of never leaving things that you couldn't fix **but** it also adds to the diff for this PR. So in practice it would have been better to not commit these files and theoretically (if you wanted to), open a second PR with these file changes. 

That's just to explain, no need to change anything now, it's 100% fine. :+1:

Reply:
I understand (and agree) why this should have been done in a second PR. 
What do you mean by "adds to the diff"?

Reply:
Sorry that isn't a terribly clear term, the "diff" corresponds to the change in the files (the difference), so if many changes are proposed then the diff is quite large and takes more work to review :)